### PR TITLE
Be less annoying when usbdk is not installed

### PR DESCRIPTION
**Pull request by leoetlino**
* Silent some warnings when usbdk is not installed. Having the user click through three warnings is not very nice.
* Make sure the user knows that USB features won't work whenever libusb fails to initialize.

### CODE REVIEW

File: Source/Core/Common/LibusbContext.cpp [hunk 1]
```diff
@@ -24,10 +25,10 @@ static libusb_context* Create()
 #ifdef _WIN32
     is_windows = true;
 #endif
-    if (is_windows && ret == LIBUSB_ERROR_NOT_FOUND)
-      PanicAlertT("Failed to initialize libusb because usbdk is not installed.");
-    else
-      PanicAlertT("Failed to initialize libusb: %s", libusb_error_name(ret));
+    const std::string reason = is_windows && ret == LIBUSB_ERROR_NOT_FOUND ?
+                                   "because usbdk is not installed" :
```
Comment (JosJuice) [inline Source/Core/Common/LibusbContext.cpp]: This isn't localizeable.

Comment (leoetlino) [inline Source/Core/Common/LibusbContext.cpp]: Fixed.

**Comment by mimimi085181** [comment]
Much better already, thanks. But maybe it should specify which devices won't work. It's Wii usb peripherals and the bluetooth passthrough that are affected. Users could think that their usb controller won't work.

And if a user does not plan to use any of those features, there's no need to install that driver. That means everytime Dolphin is started, that error message will appear. A better solution would be more much complicated. Something like "i understand, don't show this again" and save that in the ini file. And then show it again, when selecting usb passthrough. And even this would leave people not knowing why usb peripherals won't work, if they try to use them months later.

Anyways, for me this is good enough.

File: Source/Core/Common/LibusbContext.cpp [hunk 2]
```diff
@@ -25,9 +25,15 @@ static libusb_context* Create()
     is_windows = true;
 #endif
     if (is_windows && ret == LIBUSB_ERROR_NOT_FOUND)
-      PanicAlertT("Failed to initialize libusb because usbdk is not installed.");
+    {
+      PanicAlertT("Failed to initialize libusb because usbdk is not installed."
```
Comment (JosJuice) [inline Source/Core/Common/LibusbContext.cpp]: Missing space or `\n` after the period.

**Comment by JosJuice** [comment]
> And if a user does not plan to use any of those features, there's no need to install that driver. That means everytime Dolphin is started, that error message will appear.

Many people don't use those features, so this is going to get pretty annoying.

> A better solution would be more much complicated.

Maybe we could show it only if any USB devices are whitelisted or Bluetooth passthrough is enabled?

**Comment by leoetlino** [comment]
Let's see where LibusbContext::Get() gets called right at boot…

**Comment by JosJuice** [comment]
I would assume it's going to take at least a little while to get a working implementation of what I suggested, so I'm fine with merging this PR right away just to make the problem more manageable even though it isn't perfect.

(Note: I haven't reviewed the code except for LibusbContext.cpp, so this comment isn't an LGTM for the code)

**Comment by leoetlino** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**